pkgs/applications/misc/foot/default.nix

```diff
@@ -0,0 +1,38 @@
+{ stdenv, fetchgit
+, fcft, freetype, pixman, libxkbcommon, fontconfig, wayland
+, meson, ninja, ncurses, scdoc, tllist, wayland-protocols, pkg-config
```

Comment:
You didn't split each of them to it's own line. It's only a personal style preference, but IMO it's much easier to review future diffs this way. Please consider applying my request.

Reply:
I don't expect the dependencies to change that much that the indicator of githubs diff rendering what was added and removed in the line wouldn't be good enough for reviewing. Also this is the way arguments and buildInputs are generally formatted in nixpkgs so I'd opt for a similar style generally.